### PR TITLE
Wireframe

### PR DESCRIPTION
**Pull request by Artbsc1992**
# Achievements:

- Complete calculator wireframe using the class-based component.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,40 +1,9 @@
...
-          <button type="button">3</button>
-          <button type="button">+</button>
-        </div>
-        <div className="buttons line5">
-          <button type="button">0</button>
-          <button type="button">.</button>
-          <button type="button">=</button>
-        </div>
-      </div>
+      <Calculator />
```
Comment (Kingstalux) [inline src/App.js]: - [x] Good job so far 👍 As per the requirements, all your components should be class-based. Kindly convert this `app` component from a function to a class 😄

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @Artbsc1992, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f744b797263304e546c4563475764546870342f67697068792e676966


## Highlights 👏 
✅ App works as expected 🔥 
✅ Good commit history 👍 
✅ Linters are passing 👌 
✅ Correct Gitflow used 👏 


## Required Changes ♻️
_Check the comments under the review._
- [ ] Kindly make your README file more professional by removing the unused sections, adding your information to the author's section, and adding information on how a user can set up and run your project locally on their machine (`npm install`, `npm start`) 💯 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -7,50 +7,24 @@
...
 
 ## Live Demo (if available)
 
 [Live Demo Link](https://livedemo.com)
 
 
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```
Comment (TedLivist) [inline README.md]: - [x] As stated by the previous reviewer, kindly add specific instructions on how to set up and run the project locally. i.e. 
  - clone the repo
  - cd into the directory
  - run `npm install` to install packages
  - run `npm start` to run the app

File: README.md [hunk 3]
```diff
@@ -7,50 +7,24 @@
...
-
-
 ## Authors
 
-👤 **Author1**
+👤 **Arturo**
 
 - GitHub: [@githubhandle](https://github.com/githubhandle)
 - Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
Comment (TedLivist) [inline README.md]: - [x] As highlighted earlier you are expected to add the real links to your respective profiles.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Artbsc1992,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mateo951 — APPROVED** [review]
### Hi @Artbsc1992, 

Your project is complete!  There is nothing else to say other than... it's time to merge it ✔ 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**